development/compiling/compiling_with_mono.rst

```diff
@@ -375,11 +374,6 @@ the Mono module:
 
   - Build Godot with the Mono module enabled.
 
-- **mono_glue**\ =\ **yes** | no
-
-  - Whether to include the glue source files in the build
-    and define ``MONO_GLUE_DISABLED`` as a preprocessor macro.
-
 - **mono_prefix**\ =path
```

Comment:
`mono_prefix` is now an argument of the `build_assemblies.py` script, although it probably should be removed as we no longer use Mono and I feel like using Mono's MSBuild may not work.
`mono_static` and `copy_mono_root` have also been removed in 4.0.